docs/detections/rules-ui-create.asciidoc

```diff
@@ -264,7 +264,7 @@ field values.
 [[preview-rules]]
 === Preview your rule (optional)
 
-You can preview custom query, threshold, machine learning, or event correlation rules to learn how noisy a rule will be before submitting it. If necessary, you can then adjust the rule's query.
+You can preview custom query, threshold, machine learning, event correlation, or indicator match rules to learn how noisy a rule will be before submitting it. If necessary, you can then adjust the rule's query.
```

Comment:
```suggestion
You can preview a custom query, threshold, machine learning, event correlation, or indicator match rule to learn how noisy it will be before submitting it. If necessary, you can then adjust the rule's query.
```

Reply:
I think as of `8.2`, Ece said all rules will be available for preview. So maybe we can update this to: 

```suggestion
You can preview any rule to learn how noisy it will be before submitting it. If necessary, you can then adjust the rule's query.
```

Reply:
I'm a fan of shortening the sentence @jmikell821! Though, we might need to revisit this sentence in `8.3` if the new percolator feature ends up being a new rule type and doesn't have the preview ability.